frontend/src/views/c-authorship.vue

```diff
@@ -591,15 +583,16 @@ export default {
             ? this.getContributionFromAllAuthors(contributionMap)
             : contributionMap[this.info.author];
 
-        const out = {};
-        out.path = file.path;
-        out.lineCount = lineCnt;
-        out.active = lineCnt <= SINGLE_FILE_LINE_COUNT_THRESHOLD && !file.isBinary;
-        out.wasCodeLoaded = lineCnt <= SINGLE_FILE_LINE_COUNT_THRESHOLD;
-        out.fileType = file.fileType;
-        out.fileSize = file.fileSize;
-        out.isIgnored = !!file.isIgnored;
-        out.isBinary = !!file.isBinary;
+        const out: AuthorshipFile = {
+          path: file.path,
+          lineCount: lineCnt,
+          active: lineCnt <= SINGLE_FILE_LINE_COUNT_THRESHOLD && !file.isBinary,
+          wasCodeLoaded: lineCnt <= SINGLE_FILE_LINE_COUNT_THRESHOLD,
+          fileType: file.fileType,
+          fileSize: undefined, // is always undefined, to potentially remove
```

Comment:
If it is always undefined, can you check whether it is safe to remove it within this pull request